modules/cnf-configuring-workload-hints.adoc

```diff
@@ -26,3 +26,8 @@
 ----
 <1> If `highPowerConsumption` is `true`, the node is tuned for very low latency at the cost of increased power consumption.
 <2> Disables some debugging and monitoring features that can affect system latency.
+
+[NOTE]
+====
+With this `PerformanceProfile` and `realTime` set to `true` the pod specification needs to include the annotation `cpu-quota.crio.io: "disable"`.
```

Comment:
```suggestion
If `realTime` is set to `true` in a performance profile, include the `cpu-quota.crio.io annotation in the pod specification and set it to `"disable"`.
```
I do not know if this changes the technical accuracy, but I thought this might be a bit tighter.

Reply:
I'll take that thanks, guess it was not clear to me if that is just how it is set, and it should not be set to `false`.